test/website/index.html

```diff
@@ -0,0 +1,14 @@
+<!-- @format -->
```

Comment:
```suggestion
/** @format */
```

Reply:
Since this file is HTML, it needs to use the html comment syntax.